config/services/foreman.xml

```diff
@@ -2,11 +2,6 @@
 <service>
   <short>foreman</short>
   <description>Foreman is a complete lifecycle management tool for physical and virtual servers.</description>
-  <include service="dns"/>
   <include service="http"/>
   <include service="https"/>
-  <include service="dhcp"/>
-  <include service="tftp"/>
-  <port protocol="udp" port="68"/>
-  <port protocol="tcp" port="8140"/>
```

Comment:
These two ports are dropped here, but not added to any other service. They're also not mentioned in the commit. Is that expected?

Reply:
I have no clue why 68 was mentioned there. It is DHCP client, but that never made sense.
8140 is puppetserver (in firewalld still called puppetmaster).

Reply:
ACK. looks like 68 came from the original `RH-satellite-6` service.

Reply:
Do you think 68 should be added back to `RH-satellite-6`?